VirtoCommerce.JavaScriptShoppingCart.Web/Controllers/Api/JsCartCartController.cs.orig

```diff
@@ -0,0 +1,319 @@
+using System;
```

Comment:
Probably need to remove, it is temp merging file?

Reply:
Agree. This looks like a temp file

Reply:
Yes, it is right.